common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -1,53 +1,60 @@
-<form class="choicegroup capa_inputtype" id="inputtype_${id}">
-    <fieldset role="${input_type}group" aria-label="${label}">
```

Comment:
Yes @muzaffaryousaf we won't need the aria-label anymore since we can guarantee that a ```<legend>``` element will be there (line 3-7 below).  The legend serves as the accessible name.  aria-label is a way to add an accessible name to an aria widget, but since we are using a fieldset here, we want to use legend and not aria-label. 

The trick is where we get the value for the legend when dealing with legacy problems that were authored in OLX.  If the OLX label attribute cannot be matched in any of the paragraphs that precede it, we should use the value of the label attribute as the value of the legend element.  If the OLX label attribute *can* be matched to one of the preceding paragraphs, we will want to hide the paragraph from rendering in HTML. Otherwise the question text will show up twice (as a paragraph and as a legend)